.nvmrc

```
@@ -0,0 +1 @@
+v20.18.3
```

```suggestion
v20.18
```

To catch patches, probably should just be `20.18`.